sdk/imagebuilder/arm-imagebuilder/review/arm-imagebuilder.api.md

```diff
@@ -490,8 +519,8 @@ export type TriggerPropertiesUnion = TriggerProperties | SourceImageTriggerPrope
 export interface Triggers {
     beginCreateOrUpdate(resourceGroupName: string, imageTemplateName: string, triggerName: string, parameters: Trigger, options?: TriggersCreateOrUpdateOptionalParams): Promise<SimplePollerLike<OperationState<TriggersCreateOrUpdateResponse>, TriggersCreateOrUpdateResponse>>;
     beginCreateOrUpdateAndWait(resourceGroupName: string, imageTemplateName: string, triggerName: string, parameters: Trigger, options?: TriggersCreateOrUpdateOptionalParams): Promise<TriggersCreateOrUpdateResponse>;
-    beginDelete(resourceGroupName: string, imageTemplateName: string, triggerName: string, options?: TriggersDeleteOptionalParams): Promise<SimplePollerLike<OperationState<void>, void>>;
-    beginDeleteAndWait(resourceGroupName: string, imageTemplateName: string, triggerName: string, options?: TriggersDeleteOptionalParams): Promise<void>;
+    beginDelete(resourceGroupName: string, imageTemplateName: string, triggerName: string, options?: TriggersDeleteOptionalParams): Promise<SimplePollerLike<OperationState<TriggersDeleteResponse>, TriggersDeleteResponse>>;
+    beginDeleteAndWait(resourceGroupName: string, imageTemplateName: string, triggerName: string, options?: TriggersDeleteOptionalParams): Promise<TriggersDeleteResponse>;
```

Comment:
could you poin the swagger change for me? why we have this change?

Reply:
https://github.com/Azure/azure-rest-api-specs/pull/26309/commits/8c95d8c391a2a1950915b925b0957ad99d221425#diff-e35fe22422cf1b9d99241036162e4004b4772391ab86a270eaeb775a2404b577L694